ui/app/components/http-requests-bar-chart.js

```diff
@@ -89,6 +92,20 @@ export default Component.extend({
     const barChartSVG = d3.select('.http-requests-bar-chart');
     const barsContainer = d3.select('#bars-container');
 
+    // initialize the tooltip
+    const tip = d3Tip()
+      .attr('class', 'd3-tooltip')
+      .offset([HOVER_PADDING / 2, 0])
+      .html(function(d) {
+        const formatter = d3TimeFormat.utcFormat('%B %Y');
+        return `
+          <p class="date">${formatter(d.start_time)}</p>
+          <p>${Intl.NumberFormat().format(d.total)} Requests</p>
```

Comment:
Could this ever be `1 Requests`, super nitpick again (sorry!), totally not important, but I think there's an `ember-inflector` with pluralization helpers (it might be in core actually, not sure) you folks might being depending on that already?

Reply:
It is indeed [already a dependency](https://github.com/hashicorp/vault/blob/master/ui/package.json#L99). [Here](https://github.com/emberjs/ember-inflector#usage)’s the usage 😀

Reply:
these nitpicks are great! in this case, it would be EXTREMELY unlikely (potentially impossible?) to have only 1 request to a vault server since initializing and unsealing the server already makes several requests. i appreciate the tip, though!